### DISCUSSION TITLE
Why we need to export to inference model(pdmodel format)?

### DISCUSSION THREAD

**Discussion by ThanhLV97** [PaddlePaddle/PaddleOCR#6678] [Q&A] [answered]
Why we need to export to (pdmodel format) model format for prediction?

**Comment by littletomatodonkey** [comment, marked as answer]
Hi，difference between the pretrained model and inference model are as follows, you can refer to the tut for more details：https://aistudio.baidu.com/aistudio/projectdetail/4278098

<img width="714" alt="image" src="https://user-images.githubusercontent.com/14270174/175937160-f2aae98c-0c3f-49b7-a635-35b9b806d6b3.png">

**Reply by ThanhLV97** [reply]
What is the difference between prediction efficiency of pdmodel and onnx? what is the best choice?

**Reply by littletomatodonkey** [reply]
It's recommended to the pdmodel format for better efficiency according to our test results before（pdmodel is almost twice as fast since the shape info must be dealed on cpu for onnx, which is slow). You can also try to test the newest onnx version to view the change.
